editor/editor_node.cpp

```diff
@@ -409,6 +409,12 @@ void EditorNode::_unhandled_input(const Ref<InputEvent> &p_event) {
 		if (ED_IS_SHORTCUT("editor/filter_files", p_event)) {
 			filesystem_dock->focus_on_filter();
 		}
+		if (k->get_command() && k->get_keycode() == KEY_Y) {
```

Comment:
Any reason why we can't use `if (k->is_action("ui_redo", true)) {` instead?

Reply:
Wouldn't this cause the redo action to apply twice since there's also the `ui_redo` *shortcut* that is being listened to?

Reply:
Shouldn't do as the popup menu will call `accept_event`, though might need to also call that here.